adafruit_hue.py

```diff
@@ -147,12 +157,10 @@ def set_light(self, light_id, **kwargs):
         (more settings at:
         https://developers.meethue.com/develop/hue-api/lights-api/#set-light-state )
         """
-        resp = self._put(
-            "{0}/lights/{1}/state".format(self._username_url, light_id), kwargs
-        )
+        resp = self._put(f"{self._username_url}/lights/{light_id}/state", kwargs)
         return resp
 
-    def toggle_light(self, light_id):
+    def toggle_light(self, light_id: Union[int, str]) -> str:
         """Gets and toggles the current state of a specified light.
         :param int light_id: Light identifier.
```

Comment:
Same thing here, change the `int` typing in `:param:` to `int|str`

Reply:
I won't hassle you about all the others, so please make sure the documentation `:param:` typings match the newly added type annotations!